src/qml/components/KeyboardHandler.qml

```diff
@@ -12,7 +12,7 @@ Item {
     property real keyboardHeight: 0
 
     QtObject {
-        id: d
+        id: kbd
```

Comment:
Rename to the `kbd ->keyboardHandler`

Reply:
keyboardHandler is already used. kbd -> keyboardProps